PocketKit/Tests/SyncTests/Support/TestSavedItemsControllerDelegate.swift

```diff
@@ -9,10 +10,10 @@ class TestSavedItemsControllerDelegate: SavedItemsControllerDelegate {
         self.handler = handler
     }
 
-    func controllerDidChangeContent(_ controller: SavedItemsController) {
-        handler()
+    func controller(_ controller: SavedItemsController, didChange aSavedItem: SavedItem, at indexPath: IndexPath?, for type: NSFetchedResultsChangeType, newIndexPath: IndexPath?) {
     }
 
-    func controller(_ controller: SavedItemsController, didChange aSavedItem: SavedItem, at indexPath: IndexPath?, for type: NSFetchedResultsChangeType, newIndexPath: IndexPath?) {
+    func controller(_ controller: SavedItemsController, didChangeContentWith snapshot: NSDiffableDataSourceSnapshotReference) {
+        handler()
```

Comment:
`handler()` should move here. Turns out if you implement `didChangeContentWith` in NSFetchedResultsControllerDelegate `controllerDidChangeContent` is never called 🙄  thanks apple.